backends/fs/client.go

```diff
@@ -0,0 +1,63 @@
+package fs
+
+import (
+	"os"
+	"io/ioutil"
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+type Client struct {
+	rootPath    string
+	maxFileSize int64
+}
+
+func NewFsClient() (*Client, error) {
+	// TODO: how to provide rootPath in order to pass tests?
```

Comment:
The idea was to provide a simple manner to use the existing integration tests and make them work with this new backend, nothing more. 'rootPath' could be achieved with prefix anyway.